File: src/main/java/com/dyonovan/neotech/common/container/generators/ContainerFluidGenerator.java
package com.dyonovan.neotech.common.container.generators;

import com.dyonovan.neotech.common.container.BaseContainer;
import com.dyonovan.neotech.common.tileentity.generator.TileFluidGenerator;
import net.minecraft.entity.player.InventoryPlayer;
import net.minecraft.inventory.Slot;
import net.minecraft.inventory.SlotFurnaceOutput;

public class ContainerFluidGenerator extends BaseContainer {
    public ContainerFluidGenerator(InventoryPlayer inventory, TileFluidGenerator tileEntity) {
        addSlotToContainer(new Slot(tileEntity, TileFluidGenerator.BUCKET_IN, 114, 18));
        addSlotToContainer(new SlotFurnaceOutput(inventory.player, tileEntity, TileFluidGenerator.BUCKET_OUT, 114, 54));
        bindPlayerInventory(inventory, 8, 84);
    }
}

File: src/main/scala/com/dyonovan/neotech/common/container/machines/ContainerFluidGenerator.scala
package com.dyonovan.neotech.common.container.machines

import com.dyonovan.neotech.common.container.slot.SlotFurnaceOutputItemHandler
import com.dyonovan.neotech.common.tiles.machines.{TileFluidGenerator, TileElectricFurnace}
import net.minecraft.entity.player.InventoryPlayer

/**
  * This file was created for NeoTech
  *
  * NeoTech is licensed under the
  * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License:
  * http://creativecommons.org/licenses/by-nc-sa/4.0/
  *
  * @author Paul Davis <pauljoda>
  * @since 1/31/2016
  */
class ContainerFluidGenerator(playerInventory: InventoryPlayer, tile: TileFluidGenerator) extends
        ContainerAbstractMachine(playerInventory, tile) {

    addPlayerInventorySlots(8, 84)
}
